l3build.dtx

```diff
@@ -368,15 +369,15 @@
 %
 % As well as these targets, the system recognises the options
 % \begin{itemize}
-% \item |--config| (|-c|) Configuration(s) to use for testing
+% \item |--config| (|-c|) Comma or space separated list of configurations to use for testing
```

Comment:
The comma/space business is simply standard command line rules based on `--XXX` or `-X` options - I don't see that we need to mention this specifically.

Reply:
This is standard for people who know the standard. The cost of the precision is epsilon.

Reply:
I mean though that this applies to every command line tool one might use, more or less - e.g. `git` docs don't say this sort of thing

Reply:
BTW, it's not comma-or-space, it's just comma - spaces are used to separate arguments

Reply:
This is not what is implemented.
```
l3build check -e "foo bar"
```
this needs clarification either in the implementation of in the documentation

Reply:
[Lines 270-272](https://github.com/latex3/l3build/blob/5db0d64ba5647741114e3f0a7002a85fd1775bdd/l3build-arguments.lua#L270) of the `argparse` function reads
```
          for hit in gmatch(optarg, "([^,%s]+)") do
            insert(opts, hit)
          end
```
This is where argument values with spaces are managed. If spaces are not supported then the `%s` is misleading and should be removed.

Reply:
This is there to support the `--<long>[=]<option>` syntax in addition to the `-<short><option>` - you can quickly test that `--long option option2` treats `option2` as a value not an option.

Reply:
Not really,
* `--<long>=<option>` is managed at line 228
* `--<long> <option>` is managed at line 253
* `--<short><option>` is managed at line 237
id est before line 270. This line is just there to split `<option>` into a list. The separator is anything that contains a comma _or a space_.

Reply:
It was a while ago .. I will check :)

Reply:
Looking back, I used `,%s` since I added `argparse`-like syntax at all - I guess just an oversight

Reply:
I've sorted this - just commas count here